example/gardener-local/controlplane/values.yaml

```diff
@@ -162,6 +162,8 @@ global:
       MutableShootSpecNetworkingNodes: true
       WorkerlessShoots: true
     resources: {}
+    podLabels:
+      networking.resources.gardener.cloud/to-all-webhook-targets: allowed
```

Comment:
This seems to be already set when the g-API is deployed by the g-operator:
 https://github.com/gardener/gardener/blob/28df3d57178d6c2cb1f919358ccafbd64f3cc3b1/pkg/component/gardenerapiserver/deployment.go#L82C1-L82C18